lib/ebaze_web/templates/session/new.html.eex

```diff
@@ -2,4 +2,4 @@
 
 <%= render "form.html", Map.put(assigns, :action, @action) %>
 
-<span><%= link "Back", to: Routes.page_path(@conn, :index) %></span>
+<span class="button"><%= link "Back", to: Routes.page_path(@conn, :index) %></span>
```

Comment:
can the button class be applied to the link itself and not to the span around it?